installer.sh

```diff
@@ -525,11 +518,11 @@ uninstall (){
   # delete user www-embark and reset visudo
   echo -e "$ORANGE""$BOLD""Delete user""$NC"
   # sed -i 's/www\-embark\ ALL\=\(ALL\)\ NOPASSWD\:\ \/app\/emba\/emba.sh//g' /etc/sudoers #TODO doesnt work yet
-  userdel www-embark
+  userdel www-embark || true
```

Comment:
could we do a quick check if the user is on the host? like this pseudo code?
If ! grep www-embark /etc/passwd; then
userdel
fi